tests/framework/e2e/cluster.go

```diff
@@ -172,6 +172,9 @@ type EtcdProcessClusterConfig struct {
 	IsPeerAutoTLS      bool
 	CN                 bool
 
+	// ExplicitFlags contains flags to be explicitly set even if their values are same as default ones
+	ExplicitFlags map[string]struct{}
```

Comment:
Long term we might consider hardcoding differences between defaults on previous releases, however this should be a good solution for now.